presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveQueries.java

```diff
@@ -596,6 +596,8 @@ public void testIsNullIsNotNull()
     {
         assertQuery("SELECT count(*) FROM orders where clerk is not null");
         assertQuery("SELECT count(*) FROM orders where clerk is null");
+        assertQuery("select count(*) from orders_ex where quantity_by_linenumber is null");
+        assertQuery("select count(*) from orders_ex where quantity_by_linenumber is not null");
```

Comment:
`quantity_by_linenumber` is a map column. Here we see valueset (base class of sortedrangeset), I think its due to complex type which does not have ranges. Now simply passing IsNull or IsNotNull is not making the tests go through.